tools/mo/openvino/tools/mo/__init__.py

```diff
@@ -1,4 +1,4 @@
 # Copyright (C) 2018-2022 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-from .convert import convert, InputCutInfo, LayoutMap
+from .convert import convert_model, InputCutInfo, LayoutMap
```

Comment:
Please also add `convert_model` to `openvino.runtime`. We don't have much time to consider it as a separate PR. In case it is rejected during the review, we can easily remove the corresponding line. Don't do that for `InputCutInfo` and `LayoutMap` as they are not mandatory and can be replaced with raw Python syntax.